lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
The full version should show up differently. The trick is that Windows 10 will just continually be updated so behavior could be changed over time and it still be Windows 10 (with slight differences between versions).

I think as long as we have full release number as a way of tracking that, we will be fine.